documentation/book/appendix_metrics.adoc

```diff
@@ -37,64 +37,154 @@ endif::Kubernetes[]
 
-The following resources are defined:
+==== Deploying the Prometheus Operator on {OpenShiftName}
+
+To deploy the Prometheus Operator in your Kafka cluster, apply the `.yaml` files from the https://github.com/coreos/prometheus-operator/tree/master/example/rbac/prometheus-operator[Prometheus CoreOS repository].
+
+To use a different namespace than that specified in the repository files, use the following commands to download and edit the files from the repository:
+
+[source,shell,subs="+quotes"]  
+curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/rbac/prometheus-operator/prometheus-operator-deployment.yaml | sed -e 's/namespace: .*/namespace: _my-namespace_/' > prometheus-operator-deployment.yaml
+curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/rbac/prometheus-operator/prometheus-operator-cluster-role-binding.yaml | sed -e 's/namespace: .*/namespace: _my-namespace_/' > prometheus-operator-cluster-role-binding.yaml
+curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/rbac/prometheus-operator/prometheus-operator-service-account.yaml | sed -e 's/namespace: .*/namespace: _my-namespace_/' > prometheus-operator-service-account.yaml
+curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/rbac/prometheus-operator/prometheus-operator-cluster-role.yaml > prometheus-operator-cluster-role.yaml
+
+If needed, you can remove the `securityContext` from the Prometheus Operator Deployment. You can either manually remove the `spec.template.spec.securityContext` property from the `prometheus-operator-deployment.yaml` file, or run the following command:
+[source,shell,subs="+quotes"]
+yq d -i prometheus-operator-deployment.yaml spec.template.spec.securityContext
 
-* A `ClusterRole` that grants permissions to read Prometheus health endpoints of the Kubernetes system, including cAdvisor and kubelet for container metrics.  The Prometheus server configuration uses the Kubernetes service discovery feature in order to discover the pods in the cluster from which it gets metrics.  In order to have this feature working, it is necessary for the service account used for running the Prometheus service pod to have access to the API server to get the pod list.
+Next, apply all the files using the following commands:
+[source,shell,subs="+quotes"]
+oc apply -f prometheus-operator-service-account.yaml
+oc apply -f prometheus-operator-cluster-role.yaml
+oc apply -f prometheus-operator-cluster-role-binding.yaml
+oc apply -f prometheus-operator-deployment.yaml
+
+
+The {ProductName} repository contains configuration files for the Prometheus server. When you apply these configuration files, the following resources are created in your Kafka cluster and managed by the Prometheus Operator. 
```

Comment:
One sentence per line please. The first part of this para somehow doesn't fit for me here. In the second you probably want to change _Kafka cluster_ to _{OpenShiftName} cluster_?